insights/parsers/sysconfig.py

```diff
@@ -221,3 +221,26 @@ class VirtWhoSysconfig(SysconfigOptions):
         'A TEST'
     """
     pass
+
+
+@parser('sysconfig_mongod')
+class MongodSysconfig(SysconfigOptions):
+    """
+    A parser for analyzing the ``mongod`` service configuration file in
+    the ``etc/sysconfig`` directory, contains 'etc/sysconfig/mongod' and
+    '/etc/opt/rh/rh-mongodb26/sysconfig/mongod'.
+
+    Sample Input::
+
+        OPTIONS="--quiet -f /etc/mongod.conf"
+
+    Examples:
+        >>> mongod_syscfg = shared[MongodWhoSysconfig]
+        >>> mongod_syscfg['OPTIONS']
+        '--quiet -f /etc/mongod.conf'
+        >>> mongod_syscfg.get('NO_SUCH_OPTION')
+        None
+        >>> 'NOSUCHOPTION' in mongod_syscfg
+        False
+    """
+    pass
```

Comment:
I'd suggest making an `options` property here to contain the 'OPTIONS' keyword data or the empty string.  That'll make it a bit easier to get at this regularly-used piece of data.

Reply:
@PaulWay    
Currently,  `mongod_syscfg.get('OPTIONS')` works well.
Or with this `options` property, users can use `mongod_syscfg.options` instead.
I can't see any necessity there for this improvement.

And I think it's always good for user to know if an option's value is `an un-empty string` or `a empty string` or None.